app/src/lib/server/ontology.js

```diff
@@ -90,22 +90,32 @@ export async function get_version(db) {
 
 /**
  * @param {import('@cloudflare/workers-types').D1Database} db
- * @returns {(term: string) => Promise<SimplificationHint[]>}
+ * @returns {(term: string) => Promise<SimplificationHint[]>} term is case-insensitive
  */
 export const get_simplification_hints = db => async term => {
+	if (!term) {
+		return []
+	}
+
+	const ENDS_IN_A_SENSE = /-[A-Z]$/
+	if (!ENDS_IN_A_SENSE.test(term)) {
+		term = `${term}-A`
```

Comment:
This isn't right either. Now it doesn't work for words like 'accurate' that don't have a sense within the how-to document. I think we might need to change the import from the how-to document to split the stem and sense and store them separately in the db like we do in the ontology.

Reply:
fix here:  [6d53715](https://github.com/presciencelabs/tabitha-ontology/pull/25/commits/6d53715e1777d83afcd76ae5e73b5166e9c32cba)